src/selectivity-templates.js

```diff
@@ -278,6 +278,9 @@ Selectivity.Templates = {
      *                mode - Mode in which select exists, single or multiple.
      */
     selectCompliance: function(options) {
+        if (options.mode === 'multiple' && !options.name.match(/\[\]$/)) {
```

Comment:
Haha, you are right. Syntax updated, synced with current master version and built. :)